### DISCUSSION TITLE
worker received the message but is not running

### DISCUSSION THREAD

**Discussion by Seabearlu** [apache/airflow#27982] [General]
### Apache Airflow version

2.4.3

### What happened

I trigger the dag to run through the api and the task is status is queued
This is the worker run log
![image](https://user-images.githubusercontent.com/51855360/203303459-4a975594-ed35-4913-a2d1-11cb72c9102a.png)
In this regard, I found that the status of such a task in flower is active

### What you think should happen instead

worker successfully processed

### How to reproduce

It's weird, some tasks will go wrong after a while
Many of my dags have this problem
this is my cfg
![image](https://user-images.githubusercontent.com/51855360/203305070-aa734fc4-68c1-4077-b7e4-9cd0ffd0ff43.png)

### Operating System

Unbuntu20.04

### Versions of Apache Airflow Providers

airflow 2.4.1 version

### Deployment

Docker-Compose

### Deployment details

My worker and webserver are not on the same machine

### Anything else

Thanks a lot for someone who can answer

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
You have clearly problem with stability of your database/connectivity to it. You need to solve it to make Airlfow work in a stable way. Your tasks are running but then they are trying to update the state in the database and it fails. That can't work properly without stable DB connection.
